milvus/dev/load_data.py

```diff
@@ -12,19 +12,26 @@
 
 
 if __name__ == "__main__":
-    connections.connect(
-        alias="default",
-        host=os.environ["MILVUS_CLUSTER_HOST"],
-        port=os.environ["MILVUS_CLUSTER_PORT"],
-    )
-
+    if "MILVUS_API_KEY" not in os.environ or os.environ["MILVUS_API_KEY"] == "":
+        connections.connect(
+            alias="default",
+            uri=os.environ["MILVUS_CLUSTER_URI"],
+        )
+    else:
+        connections.connect(
+            alias="default",
+            uri=os.environ["MILVUS_CLUSTER_URI"],
+            token=os.environ["MILVUS_API_KEY"],
```

Comment:
Out of curiosity is it ok to connect without an API key? Does the functionality change

Reply:
The load data script can be used for both standalone and managed clusters. The standalone cluster allows connection without a key and username/password.